specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/delegatedSubnets.json

```diff
@@ -0,0 +1,515 @@
+          "patch delegated subnet": {
+            "$ref": "./examples/patchDelegatedSubnet.json"
+          }
+        },
+        "tags": [
+          "DNCDelegatedSubnet"
+        ],
+        "operationId": "DelegatedSubnetService_PatchDetails",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResourceUpdateParameters"
+            },
+            "description": "Delegated subnet details."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/DelegatedSubnet"
+            }
+          },
+          "201": {
+            "description": "InProgress. The operation is still in progress",
+            "schema": {
+              "$ref": "#/definitions/DelegatedSubnet"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "description": "Delete dnc DelegatedSubnet.",
+        "x-ms-examples": {
+          "delete delegated subnet": {
+            "$ref": "./examples/deleteDelegatedSubnet.json"
+          }
+        },
+        "tags": [
+          "DNCDelegatedSubnet"
+        ],
+        "operationId": "DelegatedSubnetService_DeleteDetails",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "202": {
+            "description": "Accepted."
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DelegatedNetwork/delegatedSubnets": {
```

Comment:
Your subscription and resource group level GET collection calls need to be marked as x-ms-pageable and include "nextLink" in their response contract.  Even if your RP isn't returning nextLinks ARM will add a nextLink if the fanout to all your regions partially fails

Reply:
This applies to all 3 resource types in this PR